src/components/CompanyEvents.vue

```diff
@@ -0,0 +1,79 @@
+<template>
+  <section class="fields">
+    <h1>{{ companyName }} Events</h1>
+    <div class="field field--superwide">
+      <EventsTable :events="events" :fetch-events="fetchEvents"></EventsTable>
+    </div>
+  </section>
+</template>
+
+<script>
+import AV from "leancloud-storage";
+import EventsTable from "@/components/EventsTable.vue";
+export default {
+  name: "CompanyEvents",
+  data() {
+    return {
+      companyName: "",
+      events: []
+    };
+  },
+  components: {
+    EventsTable
+  },
+  created() {
+    const vm = this;
+    vm.getCompanyName();
+    vm.fetchEvents();
+  },
+  methods: {
+    fetchEvents() {
+      const vm = this;
+      const clientsQuery = new AV.Query("Client");
+      clientsQuery.equalTo(
+        "company",
+        AV.Object.createWithoutData("Company", vm.$route.params.id)
+      );
+      const eventsQuery = new AV.Query("Event");
+      eventsQuery
+        .matchesQuery("client", clientsQuery)
+        .include("client")
+        .descending("createdAt")
+        .find()
+        .then(function(e) {
```

Comment:
Let's unify the way we write functions by using arrow functions only. This is just about keeping a consistent code style among the whole project.